web/styles/app_variables.css

```diff
@@ -261,7 +261,10 @@
     --left-sidebar-width: min(33.3333%, var(--left-sidebar-max-width));
     /* 40px (toggle icon) + 5px (gap) + 20px logo + 10px right margin */
     --left-sidebar-width-with-realm-icon-logo: 75px;
-    --right-sidebar-width: 250px;
+    /* 250px at 14px/1em */
+    --right-column-max-width: 17.8571em;
```

Comment:
I think this needs a second look; I would want to see this calculated at 16px/1em, though I'm not quite sure why the max width is carrying forward the old value? Isn't the goal to allow the right sidebar to occupy more than the current 250px width when there is space and a font-size for doing so?

Reply:
Set to 18em, 288px at 16px/1em